profile_collection/startup/csx1/startup/settings.py

```diff
@@ -15,7 +15,8 @@
 sd.baseline = [theta, delta, gamma,
                sx, say, saz,
                cryoangle, sy, sz,
-               epu1, epu2,
+               #epu1, epu2,
+               epu2,
```

Comment:
@wen-hu, do you want both epu's here, or just `epu2`? I think we commented the two of them for tests...